apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -8,48 +8,51 @@ import queryString from 'query-string';
 import color from '@cdo/apps/util/color';
 import $ from 'jquery';
 import {CommentArea} from './CommentArea';
-import TeacherFeedbackRubric from '@cdo/apps/templates/instructions/TeacherFeedbackRubric';
+import TeacherFeedbackKeepWorking from '@cdo/apps/templates/instructions/TeacherFeedbackKeepWorking';
 import TeacherFeedbackStatus from '@cdo/apps/templates/instructions/TeacherFeedbackStatus';
+import TeacherFeedbackRubric from '@cdo/apps/templates/instructions/TeacherFeedbackRubric';
 import {teacherFeedbackShape, rubricShape} from '@cdo/apps/templates/types';
+import experiments from '@cdo/apps/util/experiments';
 
 const ErrorType = {
   NoError: 'NoError',
   Load: 'Load',
   Save: 'Save'
 };
 
+const keepWorkingExperiment = 'teacher-feedback-review-state';
+
 export class TeacherFeedback extends Component {
   static propTypes = {
     user: PropTypes.number,
     disabledMode: PropTypes.bool.isRequired,
     rubric: rubricShape,
     visible: PropTypes.bool.isRequired,
-    //Provided by Redux
-    viewAs: PropTypes.oneOf(['Teacher', 'Student']).isRequired,
     serverScriptId: PropTypes.number,
     serverLevelId: PropTypes.number,
     teacher: PropTypes.number,
-    verifiedTeacher: PropTypes.bool,
     displayReadonlyRubric: PropTypes.bool,
     latestFeedback: teacherFeedbackShape,
     token: PropTypes.string,
+    //Provided by Redux
+    viewAs: PropTypes.oneOf(['Teacher', 'Student']).isRequired,
+    verifiedTeacher: PropTypes.bool,
     selectedSectionId: PropTypes.string
   };
 
   constructor(props) {
     super(props);
     //Pull the student id from the url
-    const studentId = queryString.parse(window.location.search).user_id;
-
+    this.studentId = queryString.parse(window.location.search).user_id;
     this.onRubricChange = this.onRubricChange.bind(this);
 
     const {latestFeedback} = this.props;
-
     this.state = {
       comment: latestFeedback?.comment || '',
-      performance: latestFeedback?.performance,
-      studentId: studentId,
+      performance: latestFeedback?.performance || null,
```

Comment:
When performance level is checked and unchecked in the DOM the performance is set to null. So for consistency performance gets initialized to null instead of undefined initially. This isn't necessary though for the component to function, so I can remove it if you'd like.

Reply:
Ok leaving this, but maybe add a short comment?